interface/weno/pharmacyHelper.php

```diff
@@ -0,0 +1,34 @@
+<?php
+/**
+ * pharmacyHelper
+ * This helper is to bridge information from the admin page ajax
+ * @package   OpenEMR
+ * @link      http://www.open-emr.org
+ * @author    Sherwin Gaddis <sherwingaddis@gmail.com>
+ * @copyright Copyright (c) 2019 Sherwin Gaddis <sherwingaddis@gmail.com>
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ *
+ */
+
+namespace OpenEMR\Rx\Weno;
+
+require_once('../globals.php');
+
+
+use OpenEMR\Pharmacy\Service\Import;
```

Comment:
Import name is not specific enough, since it may clash with other class names. Make it something more specific like ImportPharmacies

Reply:
Done